analysis/deep2_r23_o32/calibration_plots.py

```diff
@@ -209,7 +228,8 @@ def individual_gpc(individual_ascii, validation_table, log=None):
 
     log.info("starting ...")
 
-    pea_out_pdf_ind = '/Users/reagenleimbach/Desktop/Zcalbase_gal/R23O32_Manual_0417/jiang_plot_individual.pdf'
+    pea_out_pdf_ind = '/Users/reagenleimbach/Desktop/Zcalbase_gal/' \
+                       'R23O32_Manual_0417/jiang_plot_individual.pdf'
```

Comment:
Why is this still hardcoded to non-Drive path?

Reply:
Fixed it. Sorry about that.